config/settings.yml

```diff
@@ -11,3 +11,5 @@ gemfile_checker:
   pr_contacts: []
 issue_manager:
   repo_names: []
+label_notifier:
+  labelname: []
```

Comment:
well, the idea is to have multiple <labelname>: [contactlist]., so this is just an example.